configs/processors/arxiv_hybrid_v2.yaml

```
@@ -0,0 +1,134 @@
+  # Connection pooling for multiple workers
+  pool_size: 40  # Should be >= num_workers + gpu_workers + overhead
+  max_overflow: 10
+
+# ArangoDB Configuration (Expensive Computations Only)
+arangodb:
+  host: 192.168.1.69
+  port: 8529
+  username: root
+  database: academy_store
+  # password provided via command line or env var
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)